### ISSUE TITLE
Not able to find "SyslogMessageUdpSender" source class for UDP RFC 5424

### ISSUE THREAD

**Issue by arjunpidugu642** [CloudBees-community/syslog-java-client#33]
Hi Sir,

I have used your Syslog Java Client with Maven and  able to execute all sample codes except "Sample UDP sender using RFC 5424" as SyslogMessageUdpSender class is not available.

I am not even able to see SyslogMessageUdpSender class in source folder.

Please check and confirm sample code is  correct for "UDP sender using RFC 5424".

**Comment by cyrille-leclerc** [comment]
@arjunpidugu642  you are right it is `UdpSyslogMessageSender` instead of `SyslogMessageUdpSender`

https://github.com/CloudBees-community/syslog-java-client/blob/syslog-java-client-1.1.7/src/main/java/com/cloudbees/syslog/sender/UdpSyslogMessageSender.java

**Comment by cyrille-leclerc** [comment]
Please see updated documentation https://github.com/CloudBees-community/syslog-java-client
